### PR TITLE
[DOC] 3rd Party Files.txt: Add 3rdparty/ to some sdk/lib/ paths

### PR DESCRIPTION
**Pull request by SergeGautherie**
Fix wrong paths.

### CODE REVIEW

File: media/doc/3rd Party Files.txt [hunk 1]
```diff
@@ -147,7 +147,7 @@ URL: http://alter.org.ua/soft/win/uni_ata/
 
 Title: Miniport driver for the Buslogic BT 958 SCSI Controller
 Path: drivers/storage/port/buslogic
-Used Version: v1.2.0.2 (2005-08-17)
+Used Version: 1.2.0.2 (2005-08-17)
```
Comment (learn-more) [inline media/doc/3rd Party Files.txt:150]: This is VMWare's numbering scheme, not ours.

**Review by learn-more — CHANGES_REQUESTED** [review]

File: media/doc/3rd Party Files.txt (hunk 1 above)
Comment (SergeGautherie) [reply media/doc/3rd Party Files.txt]: > VMWare's numbering scheme

Really? I don't know where you got that from.

See
https://git.reactos.org/?p=reactos.git;a=blob;f=drivers/storage/port/buslogic/BusLogic958.rc;hb=HEAD#l26

Comment (learn-more) [reply media/doc/3rd Party Files.txt]: https://git.reactos.org/?p=reactos.git;a=blob;f=drivers/storage/port/buslogic/BusLogic958.c;h=76b6d8b024f8fe3614d836372326a130202ee1fb;hb=HEAD#l75

Comment (SergeGautherie) [reply media/doc/3rd Party Files.txt]: I would rather trust an active line in .rc than a comment in .c.
Also, 'v' looks like an abbreviation to me, not a numbering part.

**Review by learn-more — APPROVED** [review]

**Pull request merged**